app/webpacker/stylesheets/application.scss

```diff
@@ -0,0 +1,23 @@
+@function frontend-font-url($filename) {
+  @return url("~assets/fonts/" + $filename);
+}
+
+@function frontend-image-url($filename) {
+  @return url("~assets/images/" + $filename);
+}
+
+$govuk-font-url-function: frontend-font-url;
+$govuk-image-url-function: frontend-image-url;
+$success-green: #00823b;
+
+@import "~govuk-frontend/govuk/all";
+
+@import "admin-only";
+@import "banner";
+@import "environments";
+@import "header";
+@import "notice-summary";
```

Comment:
I don't think we need any of these from 19 down

Reply:
Removed.